src/components/functional/CalciteExpandToggle.tsx

```diff
@@ -31,13 +32,23 @@ export function toggleChildActionText({
   parent.querySelectorAll("calcite-action").forEach((action) => (action.textEnabled = expanded));
 }
 
-const setTooltipReference = (
-  tooltip: HTMLCalciteTooltipElement,
-  referenceElement: HTMLCalciteActionElement,
-  expanded: boolean
-): HTMLCalciteActionElement => {
-  if (tooltip) {
-    tooltip.referenceElement = !expanded && referenceElement;
+const setTooltipReference = ({
+  tooltipExpand,
+  referenceElement,
+  expanded,
+  ref
+}: {
+  tooltipExpand: HTMLCalciteTooltipElement;
```

Comment:
Nitpick: What's the `Expand` part of `tooltipExpand`? `tooltip` seems simpler, but I could be missing something.

Reply:
renamed these confusing props :)